storage/src/main/java/tech/pegasys/teku/storage/server/Database.java

```diff
@@ -70,7 +71,11 @@
   @MustBeClosed
   Stream<DepositsFromBlockEvent> streamDepositsFromBlocks();
 
+  Optional<ProtoArray> getProtoArrayFromDisk();
```

Comment:
nit: `fromDisk` is implied given it's a database - probably just `getProtoArray`